src/components/UserProfile/CavaticaTokenInput.js

```diff
@@ -0,0 +1,164 @@
+import * as React from 'react';
+import { compose, withState } from 'recompose';
+
+import step2Screenshot from 'assets/cavaticaTokenScreenshot.png';
+import { deleteSecret, setSecret } from 'services/secrets';
+import { CAVATICA } from 'common/constants';
+import { getUser as getCavaticaUser } from 'services/cavatica';
+import { ModalFooter } from 'components/Modal/index.js';
+
+import { css } from 'emotion';
+import { injectState } from 'freactal';
+import RightArrows from 'react-icons/lib/fa/angle-double-right';
+
+const styles = css`
+  span.numberBullet {
+    color: white;
+    background: #00afed;
+    width: 14px;
+    border-radius: 50%;
+    margin: 20px;
+    padding: 10px;
+    height: 14px;
+    display: inline-block;
+    text-align: center;
+  }
+
+  .tokenTitle {
+    padding: 20px;
+    color: #2b388f;
```

Comment:
I might be the only one using it atm, but I have the colors defined in an emotion theme https://github.com/kids-first/kf-portal-ui/blob/master/src/theme/defaultTheme.js#L5 you can use it like
```javascript
import styled from 'react-emotion';
[...]
const numberBullet = styled.span`
  color: ${props => props.theme. secondary};
`
```

Reply:
Thank you! Wasn't sure how to do this properly, so just hacked in the colour there.